### PR TITLE
Frontend feedback page

### PR DESCRIPTION
**Pull request by Elena-Erganyan**

### CODE REVIEW

File: client/src/redux/api/feedbackApi.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
-const feedbackApi = api.injectEndpoints({
+export const feedbackApi = api.injectEndpoints({
   endpoints: (builder) => ({
-    getFeedbackList: builder.query<IFeedback[], {key: string, value: string}>({
+    getFeedbackList: builder.query<IFeedback[], QueryParams>({
       query: ({key, value}) => `feedback?${key}=${value}`,
       keepUnusedDataFor: 300,
-      providesTags: (_result, _error, arg)=> [{ type: 'Feedback', id: 'LIST' }],
+      providesTags: (_result, _error, arg)=> [{ type: 'Feedback', id: 'List' }],
```
Comment (ALI-KHALED-13) [inline client/src/redux/api/feedbackApi.ts:8]: this tag will be overridden by mine.. just saying

File: client/src/components/Comment/index.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                    @{comment.parentComment.owner.username} 
+                  </StyledCommentReplyName>
+                : null
+              )}
+              {comment.content}
+            </StyledCommentText>
+          </StyledComment>
+      }
+
+      {isReplyFormShown &&
```
Comment (ALI-KHALED-13) [inline client/src/components/Comment/index.tsx:154]: I believe this form can be unified with the from in FeedbackPage, all joined under 1 component?

**Review by ALI-KHALED-13 — CHANGES_REQUESTED** [review]
few comments

File: client/src/pages/FeedbackPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+          description={`Comment on the feedback which was made to ${productData?.title}`}
+        />
+        <AlertMessage
+          state="error"
+          enabled={Boolean(feedbackError || commentsError || productError)}
+          text={getErrorMessage(feedbackError || commentsError || productError) as string}
+          duration={10000}
+        />
+
+        <StyledFeedbackPageWrapper>
```
[the remark this answers is not in the document]
Comment (Elena-Erganyan) [reply client/src/pages/FeedbackPage/index.tsx:123]: Agreed, will fix it
I just used this component before refactoring in several places on the page, that's way made up so 'general' name

**Review by ALI-KHALED-13 — APPROVED** [review]
Well done :)

**Pull request merged**